model/registry/test.yaml

```diff
@@ -0,0 +1,72 @@
+groups:
+  - id: registry.test.suite
+    prefix: test.suite
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to
+     [software tests](https://en.wikipedia.org/wiki/Software_testing).
+    attributes:
+      - id: name
+        type: string
+        stability: experimental
+        brief: >
+          The human readable name of a [test suite](https://en.wikipedia.org/wiki/Test_suite).
+        examples: ['TestSuite1']
+      - id: run.id
```

Comment:
Totally a good and valid question. A test  `suite` is a group of related test cases. My interpretation is the term test class is mostly a language specific term. 

It's very likely that `test.suite.run.id` is the same as `cicd.pipeline.run.id` but it's possible this is not a guarantee. For example, we have folks that run tests locally as part of the pre-commit workflow & there's an opt-in feature that allows them to send those tests within a trace to the platform gateway collectors. It could be that they run just a suite of tests, one at a time, which could warrant it's own `run.id`.

However, @jsuereth  brought up a good point about things living forever once they get merged in, so I would be fine with removing `run.id`'s from the test suite in this iteration & opening a ticket for further discussion if that's what makes the most sense.

Reply:
Sounds good! If we don't have an immediate use-case for it. let's not add it just yet

Reply:
Have removed and added issue #1262 to discuss further.